src/App.tsx

```diff
@@ -1,14 +1,55 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isTodoSelected, setIsTodoSelected] = useState(false);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
```

Comment:
`selectedTodo` can be used only. `isTodoSelected` isn't necessary
```suggestion
  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
  const isTodoSelected = !!selectedTodo;
```